charts/govuk-rails-app/templates/nginx-configmap.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.nginxConfigMap.create }}
```

Comment:
Currently when an app needs an Nginx config it can use a custom one by specifying `.Values.nginxConfigMap.name: my-custom-configmap`, or it can use the one built into the `govuk-rails-app` chart by setting `.Values.nginxConfigMap.create: true` (the default).

Perhaps we should move this configmap to the `govuk-apps-conf` Helm chart, so we have a consistent pattern for providing Nginx configs to apps. What do you think?

Reply:
We might run in to issues if we move the config map into govuk-apps-conf, since we lose the ability to template in values for that specific application (things like app port would have to be static). If we don't need that, it would probably make sense to move it

Reply:
You're right, let's leave it here - thanks!

Reply:
I don't think it's possible to move this configmap (the one that's being created for most apps by govuk-rails-conf chart at the moment) into govuk-apps-conf chart. Each app needs its own nginx configmap. If you meant giving every app its own separate configmap in govuk-apps-conf chart, then I don't think that buys us anything and just adds complexity.

Reply:
Yep makes sense @sengi, I didn't realise that each app needed a unique nginx configmap.